packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -144,6 +151,31 @@ const TradingAppCard = ({
                             text={badge_text}
                         />
                     )}
+                    {is_open_order_position_status_present && (
+                        <StatusBadge
+                            className='trading-app-card__acc_status_badge'
+                            onClick={() => {
+                                setModalOpen(!is_modal_open);
+                            }}
+                            account_status='open-order-position'
```

Comment:
for what we use this prop "account_status" here?

Reply:
since its a common component, there are some classes that use the value of status flag to append at the end. So, basically the class changes based on the value of the flag. For eg- `switcher-status-badge--pending` gives yellow shade `--failed` would give red and so on.

Reply:
`account_status` is giving this `--pending, --open-order-position` to classes to render accordingly.